File: 2. Leetcode Solutions/0344-reverse-string/0344-reverse-string.cpp

class Solution {
public:
    void reverseString(vector<char>& s) {
        int seedhe = 0;
        int maut = s.size() - 1;
        while (seedhe < maut) {
            // swapp
            char temp = s[seedhe];
            s[seedhe] = s[maut];
            s[maut] = temp;
            seedhe++;
            maut--;
        }
    }
};

File: 2. Leetcode Solutions/0344-reverse-string/0344-reverse-string.py
class Solution(object):
    def reverseString(self, s):
        seedhe = 0
        maut = len(s) - 1
        while seedhe < maut:
            s[seedhe], s[maut] = s[maut], s[seedhe]  # Swap
            seedhe += 1
            maut -= 1